zomes/hc_zome_trust_atom/src/lib.rs

```diff
@@ -16,7 +16,10 @@ use std::collections::BTreeMap;
 // public for sweettest; TODO can we fix this:
 pub mod trust_atom;
 pub use crate::trust_atom::*;
-pub mod test_helpers;
+pub mod trust_graph;
+pub use crate::trust_graph::*;
+pub(crate) mod utils;
+pub(crate) mod test_helpers;
```

Comment:
can sweetest still use this?

Reply:
tied using pub(super) got error: "too many leading super keywords"

going to leave it public for now